rpc/api.proto

```diff
@@ -186,6 +200,27 @@ message GetTransactionsResponse {
 	repeated TransactionDetails unmined_transactions = 2;
 }
 
+message TicketPriceRequest {
+}
+message TicketPriceResponse {
+	int64 ticket_price = 1;
+	int32 height = 2;
+}
+
+message StakeInfoRequest {
+}
+message StakeInfoResponse {
+	uint32 pool_size = 1;
+	uint32 all_mempool_tix = 2;
+	uint32 own_mempool_tix = 3;
+	uint32 immature = 4;
+	uint32 live = 5;
+	uint32 voted = 6;
+	uint32 missed = 7;
+	uint32 revoked = 8;
```

Comment:
Should there be a placeholder for expired tickets?
https://forum.decred.org/threads/one-of-my-oldest-tickets-rip.3873/#post-21184